docs/monitoring/08-03-define-alerting-rules.md

```diff
@@ -0,0 +1,75 @@
+---
+
+This tutorial shows you how to define alerting rules to monitor the health status of your resources. In this example, you will write an alerting rule based on the `cpu_temperature_celsius` metric. The alert defined in the rule will fire whenever the CPU temperature exceeds 70 degrees Celsius.
+
+## Prerequisites
+
+This tutorial is a follow-up of the [Observe application metrics](components/monitoring/#tutorials-expose-custom-metrics-in-kyma) tutorial that uses the `monitoring-custom-metrics` example. Follow this tutorial to deploy the `sample-metrics-8081` service which exposes the `cpu_temperature_celsius` metric. That configuration is required to complete this tutorial.
+
+## Steps
+
+Follow these steps to create an alerting rule:
+
+1. Create a PrometheusRule resource holding the configuration of your alerting rule. 
+
+>**NOTE**: Prometheus requires a specific label to identify PrometheusRule definitions. Make sure you set **role** to `alert-rules`.
+
+```yaml
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: cpu.temp.rules
+  labels:
+    app: cpu.temp.rules
+    prometheus: monitoring
+    release: monitoring
+    role: alert-rules
+spec:
+  groups:
+  - name: cpu.temp.rules
+    rules:
+    - alert: CPUTempHigh
+      expr: cpu_temperature_celsius > 70 
+      for: 60s
+      labels:
+        severity: critical
+      annotations:
+        description: "CPU temperature exceeds 70 degrees Celsius"
+        summary: "CPU temperature is too high"
+```
+Configure your alert rule using the following parameters:
+
+| Parameter | Description | Example value |
+|-----------|-------------|---------------|
+| **groups.name** | Specifies the name of the group listing the rules.  | `cpu.temp.rules` |
+| **rules.alert** | Specifies the name of the alert. | `CPUTempHigh`  |
+| **rules.expr** | A PromQL expression specifying the conditions that must be met for the alarm to fire. Specify the expression using Kubernetes [functions](https://prometheus.io/docs/prometheus/latest/querying/functions/) and [metrics](https://github.com/kubernetes/kube-state-metrics/blob/master/Documentation/pod-metrics.md). | `cpu_temperature_celsius > 70`  |
+| **rules.for** | Specifies the time period between encountering an active alert for the first time during rule evaluation and firing the alert.  | `60s` |
+| **rules.labels.severity** | Specifies the severity of the alert.  | `critical` |
+| **rules.annotations.description** | Provides the alert details. | `CPU temperature exceeds 70 degrees Celsius` |
+| **rules.annotations.summary** | Provides a short alert summary. | `CPU temperature is too high` |
+
+
+
+For more details on defining alerting rules, see [this](https://prometheus.io/docs/prometheus/latest/configuration/alerting_rules/) document.
+
+3. Deploy the alerting rule:
+
+```bash
+kubectl apply -f test-rules.yaml
+```
+4. Run the port-forward command on the monitoring-prometheus service to access the Prometheus dashboard:
```

Comment:
DO we use any kind of formatting on commands, services, and dashboards names? 🤔

Reply:
I formatted the command and the service in this case.